upgrade/patches/41_custom_field_required.sql

```diff
@@ -0,0 +1,3 @@
+-- Add field to allow custom fields to be marked as required when editing
+alter table {{%custom_field}}
+    add `fld_edit_form_required` tinyint(1) NOT NULL DEFAULT 0;
```

Comment:
yes, this is just for new fields.

i think i will recreate schema at some point, marking all applied patches applied, and there can do some reordering fun. or you think it's worth to do patch as well?

and do you have opinion how to order int fields? maybe some internal info? ;)